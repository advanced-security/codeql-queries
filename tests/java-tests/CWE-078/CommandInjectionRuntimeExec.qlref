CWE-078/CommandInjectionRuntimeExecTest.ql