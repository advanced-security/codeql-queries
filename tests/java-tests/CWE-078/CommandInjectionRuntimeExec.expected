| src/main/java/com/github/advanced_security/command_injection_test/Main.java:22:39:22:51 | commandArray1 | Call to dangerous java.lang.Runtime.exec() with command '$@' with arg from untrusted input '$@' | src/main/java/com/github/advanced_security/command_injection_test/Main.java:21:51:21:59 | "/bin/sh" | "/bin/sh" | src/main/java/com/github/advanced_security/command_injection_test/Main.java:16:29:16:41 | args | args |
| src/main/java/com/github/advanced_security/command_injection_test/Main.java:30:39:30:51 | commandArray2 | Call to dangerous java.lang.Runtime.exec() with command '$@' with arg from untrusted input '$@' | src/main/java/com/github/advanced_security/command_injection_test/Main.java:26:32:26:40 | "/bin/sh" | "/bin/sh" | src/main/java/com/github/advanced_security/command_injection_test/Main.java:16:29:16:41 | args | args |
| src/main/java/com/github/advanced_security/command_injection_test/Main.java:34:17:37:40 | toArray(...) | Call to dangerous java.lang.Runtime.exec() with command '$@' with arg from untrusted input '$@' | src/main/java/com/github/advanced_security/command_injection_test/Main.java:35:48:35:56 | "/bin/sh" | "/bin/sh" | src/main/java/com/github/advanced_security/command_injection_test/Main.java:16:29:16:41 | args | args |
